attachments/templates/attachments/bootstrap/form.html

```diff
@@ -0,0 +1,5 @@
+{% extends "attachments/form.html" %}
+{% load bootstrap %}
+{% block render %}
+    {% bootstrap_form form %}
+{% endblock render %}
```

Comment:
The only thing in the parent template is the single block render... I wonder if it's even necessary to extend that template? Or just remove the blocks all together? Adding the extends templatetag is just added unnecessary overhead

```django
{% load bootstrap %}
{% bootstrap_form form %}
```

```django
{{ form }}
```

Reply:
Yeah removing them makes sense. If the user wants to overwrite the templates and add more stuff they will still be able to add the extends and blocks if they have a use for them